sphinx/source/docs/install.md

```diff
@@ -286,10 +287,30 @@ source install/setup.bash
 
     Ubuntu 22.04 sysroot is not fully prepared for cross-compilation (but for native builds instead) and thereby, while invoking FindPython, it's just picking the host resources, instead of the target/sysroot ones, which leads to ROS 2 packages relying on Python 3 getting a dependency against the host (which doesn't exist), instead of against the sysroot.
 
-    A **workaround** for this is symlinking the Python3 library of the host to the sysroot one, so that it gets picked while cross-compiling against the Ubuntu 22.04 sysroot. The following should do: `sudo ln -s ~/krs_ws/src/install/../acceleration/firmware/kr260/sysroots/aarch64-xilinx-linux/usr/lib/aarch64-linux-gnu/libpython3.10.so.1.0 /usr/lib/aarch64-linux-gnu/libpython3.10.so`)
+    A **workaround** for this is symlinking the Python3 library of the host to the sysroot one, so that it gets picked while cross-compiling against the Ubuntu 22.04 sysroot. The following should do: `sudo ln -s ~/krs_ws/install/../acceleration/firmware/kr260/sysroots/aarch64-xilinx-linux/usr/lib/aarch64-linux-gnu/libpython3.10.so.1.0 /usr/lib/aarch64-linux-gnu/libpython3.10.so`)
 ```
 
 ``` shell
+###################################################
+# 6.B workaround for KR260 additional firmware elements
+#   see https://github.com/Xilinx/KRS/issues/62#issue-1292007180
+#
+# TODO: remove this once these artifacts are added to KR260
+#   firmware ROS package (acceleration_firmware_kr260)
+###################################################
+# fetch KV260 firmware
+cd ~/krs_ws
+wget https://www.xilinx.com/bin/public/openDownload?filename=acceleration_firmware_kv260.zip -P src/firmware/acceleration_firmware_kv260
```

Comment:
@jasvinderkhurana because kr260 firmware is missing some artifacts which don't allow to build accelerators. We're fetching those from the KV260 (same SoC and SOM, so artifacts work the same).

A future release of acceleration_firmware_kr260 should address that completely.

Reply:
For completeness, dropping in here the original source wherein we detected the need to release another firmware release: https://github.com/Xilinx/KRS/issues/62#issue-1292007180

Next one should be out in early Sept.